.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

Comment:
Maybe we should add .idea and .vscode to the gitignore instead of checking them in.

Reply:
Yup, in the TODO list.